targets/ath79-generic

```diff
@@ -123,6 +123,12 @@ device('tp-link-archer-d50-v1', 'tplink_archer-d50-v1', {
 	broken = true, -- 64M ath9k + ath10k & power LED not working
 })
 
+device('tp-link-cpe210-v1', 'tplink_cpe210-v1', {
+	manifest_aliases = {
+		'tp-link-cpe210-v1.0',
```

Comment:
Please add a comment to these manifest aliases that they are only used for upgrades from OpenWrt 19.07, so we know that we can get rid of them in the future.

Reply:
Resolved in b2dd511785b611e612d626f14a2379d36cc77d3a, I think.

Reply:
The Gluon version in the comment is wrong, the last version for which the manifest alias is relevant is 2021.1, not 2020.1. Or just remove it and only mention the OpenWrt version.

Also, I'd prefer a short comment for each manifest alias that can be removed in the future, maybe like https://github.com/freifunk-gluon/gluon/pull/2417#issuecomment-1076863818, so we don't need to guess which aliases the comment refers to when we look at it in a few years.

Reply:
You are right, I looked for the first and not the last gluon version.

Resolved in 440fb7032256317889942e09341b3790a5596180.